### PR TITLE
[easy] Small reorg of SnappBase code

### PR DESCRIPTION
**Pull request by fleupold**
1) Putting all view methods together (removing `isDepositSlotEmpty`, since it can be derived from `getDepositHash`)
2) Small changes to comments

### CODE REVIEW

File: contracts/SnappBase.sol [hunk 1]
```diff
@@ -270,13 +287,4 @@ contract SnappBase is Ownable {
         // There is no situation where contract balance can't afford the upcomming transfer.
         ERC20(tokenIdToAddressMap[tokenId]).transfer(accountToPublicKeyMap[accountId], amount);
     }
-
-    function getDepositCreationBlock(uint slot) public view returns (uint) {
```
Comment (josojo) [inline contracts/SnappBase.sol:274]: Can we please leave them in for now? I could not find a way to read struct parameters with the web3 rust library. In js it's easy, but with rust, I was not finding any solution for 2 days...

Once, we have a solution, I am perfectly fine with removing these lines

**Review by josojo — CHANGES_REQUESTED** [review]
Besides removing the "stupid rust-helper functions", it looks all good

**Review by josojo — APPROVED** [review]
I have overseen that you just moved the view funcitons. All good

**Review by bh2smith — APPROVED** [review]
Looks fantastic! Thanks

**Pull request merged**